tools/clang/unittests/HLSL/CompilerTest.cpp

```diff
@@ -231,7 +231,10 @@ class CompilerTest : public ::testing::Test {
   TEST_METHOD(CompileWhenODumpThenOptimizerMatch)
   TEST_METHOD(CompileWhenVdThenProducesDxilContainer)
 
+#if _ITERATOR_DEBUG_LEVEL==0 
+  // CompileWhenNoMemThenOOM can properly detect leaks only when debug iterators are disabled
   TEST_METHOD(CompileWhenNoMemThenOOM)
```

Comment:
Another way to do this would be to set Ignore to true for this method in this case.  Then the test would still be present, just not run unless explicitly selected and you used /runIgnoredTests.
See:
`TEST_METHOD_PROPERTY(L"Ignore", L"true")`

Reply:
Yes, but this way it will always be run when possible and we don't have to make sure to explicitly run it with /runIgnoredTests, or to add it to hcttest.